app/lib/actions/katello/organization/environment_contents_refresh.rb

```diff
@@ -0,0 +1,22 @@
+module Actions
+  module Katello
+    module Organization
+      class EnvironmentContentsRefresh < Actions::AbstractAsyncTask
+        middleware.use Actions::Middleware::PropagateCandlepinErrors
+
+        def plan(organization_id)
+          ::Katello::ContentViewEnvironment.all.select do |cve|
```

Comment:
Why not
```ruby
organization = Organization.find(organization_id) # or better off pass the organization object.
organization.content_view_environments.each do |cve|
   plan_action(
            Actions::Candlepin::Environment::SetContent,
            cve.content_view,
            cve.owner,
            cve,
     )
end

```

Reply:
Tried this out and it works nicely. Thanks!